app/views/home/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :fluid_body do %>
   <%= tag.div class: 'hero', style: "background-image: url(#{image_path(@hero_image)});" do %>
-    <div class="row align-items-center justify-content-center h-100 gy-2">
-      <div class="col-md-3 col-sm-12">
+    <div class="row align-items-center justify-content-center h-100 gy-2" role="list">
```

Comment:
https://dequeuniversity.com/class/semantic-structure/lists

Reply:
this is a cool idea! I've never used `role=list` on a series of cards before, but it makes total sense – this is just a list of content that happens to be visually represented as cards.